content/docs/setup/kubernetes/istio-cni-install/index.md

```diff
@@ -0,0 +1,175 @@
+
+    1.  Refer to the [Hosted Kubernetes Usage](#hosted-kubernetes-usage) section for any non-default
+        settings required.
+
+1.  Add the `--set istio_cni.enabled=true` setting to the [Istio Installation with Helm procedure](/docs/setup/kubernetes/helm-install/) to include the installation of the Istio CNI plugin. For example:
+
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+      --set istio_cni.enabled=true > $HOME/istio.yaml
+    {{< /text >}}
+
+    > Refer to the full set of `istio-cni` Helm parameters in the chart's [`values.yaml`](https://github.com/istio/cni/blob/master/deployments/kubernetes/install/helm/istio-cni/values.yaml).
+
+### Example: Excluding specific Kubernetes namespaces
+
+The following example configures the Istio CNI plugin to ignore pods in the namespaces `istio-system`,
+`foo_ns`, and `bar_ns`.
+
+1.  Create an Istio manifest with the Istio CNI plugin enabled and override the default
+    configuration of the `istio-cni` Helm chart's `logLevel` and `excludeNamespaces` parameters:
+
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --set istio_cni.enabled=true \
+    --set istio-cni.logLevel=info \
+    --set istio-cni.excludeNamespaces={"istio-system,foo_ns,bar_ns"} > $HOME/istio.yaml
+    {{< /text >}}
+
+### Helm chart parameters
+
+| Option | Values | Default | Description |
+|--------|--------|---------|-------------|
+| `hub` | | | The container registry to pull the `install-cni` image. |
+| `tag` | | | The container tag to use to pull the `install-cni` image. |
+| `logLevel` | `panic`, `fatal`, `error`, `warn`, `info`, `debug` | `warn` | Logging level for CNI binary. |
+| `excludeNamespaces` | `[]string` | `[ istio-system ]` | List of namespaces to exclude from Istio pod check. |
+| `cniBinDir` | | `/opt/cni/bin` | Must be the same as the environment's `--cni-bin-dir` setting (`kubelet` parameter). |
+| `cniConfDir` | | `/etc/cni/net.d` | Must be the same as the environment's `--cni-conf-dir` setting (`kubelet` parameter). |
+| `cniConfFileName` | | None | Leave unset to auto-find the first file in the `cni-conf-dir` (as `kubelet` does).  Primarily used for testing `install-cni` plugin configuration.  If set, `install-cni` will inject the plugin configuration into this file in the `cni-conf-dir`. |
+
+### Hosted Kubernetes Usage
+
+Not all hosted Kubernetes clusters are created with the `kubelet` configured to use the CNI plugin so
+compatibility with this `istio-cni` solution is not ubiquitous.  The `istio-cni` plugin is expected
+to work with any hosted Kubernetes leveraging CNI plugins.  The below table indicates the known CNI status
+of many common Kubernetes environments.
+
+| Hosted Cluster Type | Uses CNI | Required Non-default settings |
+|---------------------|----------|-------------------------------|
+| GKE 1.9+ default | N | |
```

Comment:
alphabetize list please.  Over time we will want to rework this into the platfrorm section documentation.